_topic_map.yml

```
@@ -186,6 +186,8 @@ Topics:
   File: integrate-with-image-vulnerability-scanners
 - Name: Integrating with Jira
   File: integrate-with-jira
+- Name: Integrating with Email
```

Per ISG and to match sentence case, I think email here should be lowercase
```suggestion
- Name: Integrating with email
```